java/src/main/java/com/logicalclocks/hsfs/FeatureStore.java

```diff
@@ -54,6 +55,7 @@ public class FeatureStore {
   private TrainingDatasetApi trainingDatasetApi;
   private StorageConnectorApi storageConnectorApi;
   private ExpectationsApi expectationsApi;
+  private FeatureViewEngine featureViewEngine;
 
   private static final Logger LOGGER = LoggerFactory.getLogger(FeatureStore.class);
 
```

Comment:
I know this has been like it all along, but I wonder why we make the default version 1 and not the "latest". This might probably either require an additional call to the backend (to check for the latest verson number (less favourable)) or a change in the backend (to make it undertand some kind of "latest" flag). To me this behaviour would seem more intuitive. If I want to make sure I always receive the same version, I specify the version number explicitly. And if I don't I simple receive the "latest" one. Why would I by default want the "oldest" one?

Reply:
maybe @moritzmeister can weight in?